docs/modules/publisher-migrationpath.md

```diff
@@ -179,17 +161,17 @@ When there are configured endpoints:
 }
 ```
 
-### GetConfiguredNodesOnEndpoint_V1
+### GetConfiguredNodesOnEndpoint (GetConfiguredNodesOnEndpoint_V1)
 
-`Request`:
+`Request`
 
 ```json
 {
    "EndpointUrl": "opc.tcp://sandboxhost-637811493394507132:50000"
```

Comment:
Please add an example with all parameters and fix the corresponding explanation "..the other attributes are used to refine the result..."

Reply:
Fixed description of request for `GetConfiguredNodesOnEndpoint_V1`. Added full example.